env.default

```diff
@@ -58,6 +58,15 @@ BASKET_SQS_QUEUE_URL=
 # See the Basket client docs https://basket-client.readthedocs.io/en/latest/install.html
 BASKET_API_ROOT_URL=https://basket-dev.allizom.org/
 
+# CSP config
+CSP_CONNECT_SRC=" * "
+CSP_FONT_SRC=" 'self' fonts.googleapis.com fonts.gstatic.com"
+CSP_FORM_ACTION=" 'self' https://webto.salesforce.com/ mozillasalescloud--mkl.cs4.my.salesforce.com "
+CSP_FRAME_SRC=" 'self' assets.braintreegateway.com c.paypal.com *.paypal.com https://www.google.com/recaptcha/api2/ https://tst.kaptcha.com/ "
+CSP_IMG_SRC=" * data: "
+CSP_SCRIPT_SRC=" 'self' 'unsafe-inline' https://www.google-analytics.com/analytics.js  https://www.google-analytics.com/plugins/ua/ecommerce.js js.braintreegateway.com assets.braintreegateway.com www.paypalobjects.com c.paypal.com www.paypal.com https://www.google.com/recaptcha/api.js https://www.gstatic.com/recaptcha/releases/" 
```

Comment:
Most of these look sensible, although we want some https:// prefixes for those bare domains.